src/ontodia/data/utils.ts

```diff
@@ -0,0 +1,27 @@
+import * as N3 from "n3";
+import { RdfNode, Triple } from './sparql/sparqlModels';
+
+export function N3toRdfNode(entity: string): RdfNode {
+    if (entity.length >= 2 && entity[0] === '"' && entity[entity.length - 1] === '"') {
```

Comment:
This won't work for n3 literals with specified datatype, like this: `"42"^^<http://www.w3.org/2001/XMLSchema#integer>`